rules/windows/powershell/powershell_script/posh_ps_cl_invocation_lolscript.yml

```diff
@@ -1,4 +1,4 @@
-title: Execution via CL_Invocation.ps1
+title: Execution via CL_Invocation.ps1 - Powershell Script Block Text
```

Comment:
`title: Execution via CL_Invocation.ps1 - PowerShell`
Like the other rules

Reply:
Modified